src/bundle/ui-dev/src/modules/universal-discovery/services/universal.discovery.service.js

```diff
@@ -1,6 +1,7 @@
 import { getRequestHeaders, getRequestMode } from '../../../../../Resources/public/js/scripts/helpers/request.helper.js';
 import { showErrorNotification } from '../../common/services/notification.service';
 import { handleRequestResponse, handleRequestResponseStatus } from '../../common/helpers/request.helper.js';
+import { getAdminUiConfig } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/context.helper';
```

Comment:
I'm little torn, as you used here bundle aliast, but in line 1 you import sth else from helpers, so maybe change first line to use alias as well, pretty please? ;)
Also, I would group this one with the one in first line, as they both come from same place, unlike the ones in lines 2 and 3

Reply:
As this is a bug fix, I don't have to refactor the old code, and we are talking about organising imports here.
Nonetheless, I will adjust them.